src/content/docs/kubernetes-pixie/kubernetes-integration/advanced-configuration/configure-control-plane-monitoring.mdx

```diff
@@ -0,0 +1,522 @@
+
+## Compatibility and requirements [#compatibility]
+
+* Control plane monitoring support is limited for managed clusters. This is because most cloud providers do not expose the metrics endpoints for the control plane components, so New Relic cannot access them.
+* When deploying the solution in [unprivileged mode](/docs/integrations/kubernetes-integration/installation/kubernetes-installation-configuration#unprivileged), control plane setup will require [extra steps](#hostnetwork-privileged) and some caveats might apply.
+* [OpenShift](http://learn.openshift.com/?extIdCarryOver=true&sc_cid=701f2000001OH7iAAG) 4.x uses control plane component metric endpoints that are different than the default.
+
+## Control plane component [#component]
+
+The task of monitoring the Kubernetes control plane is a responsibility of the `nrk8s-controlplane` component, which by default is deployed as a DaemonSet. This component is automatically deployed to master nodes, through the use of a default list of `nodeSelectorTerms` which includes labels commonly used to identify master nodes, such as `node-role.kubernetes.io/control-plane` or `node-role.kubernetes.io/master`. Regardless, this selector is exposed in the `values.yml` file and therefore can be reconfigured to fit other environments.
+
+Clusters that do not have any node matching these selectors will not get any pod scheduled, thus not wasting any resources and being functionally equivalent of disabling control plane monitoring altogether by setting `controlPlane.enabled` to `false` in the Helm Chart.
+
+Each component of the control plane has a dedicated section, which allows to individually:
+
+* Enable or disable monitoring of that component
+* Define specific selectors and namespaces for discovering that component
+* Define the endpoints and paths that will be used to fetch metrics for that component
+* Define the authentication mechanisms that need to be used to get metrics for that component
+* Manually specify endpoints that skip autodiscovery completely
+
+### Autodiscovery and default configuration [#autodiscovery-default]
+
+By default, our [Helm Chart](/docs/kubernetes-pixie/kubernetes-integration/installation/install-kubernetes-integration-using-helm) ships a default configuration that should work out of the box for some control plane components for on-premise distributions that run the control plane inside of the cluster, such as `Kubeadm` or `minikube`.
+
+#### `hostNetwork` and `privileged` [#hostnetwork-privileged]
+
+Most users and Kubernetes distributions configure the control plane metrics endpoints to listen only in the loopback interface, i.e. `localhost`. For this reason, the control plane component is deployed with `hostNetwork: true` by default when `privileged` is set to `true` (the default).
```

Comment:
"listen only _to_ the loopback interface"?

Reply:
From what I know, when talking about IP addresses (or interfaces, which is mostly the same) it is common to use listen _on_ rather than _to_. But I haven't found any "official guidance" on which is more correct.